### PR TITLE
Greening: Master

### PR DESCRIPTION
**Pull request by IhorBohdan**

### CODE REVIEW

File: cypress/integration/invoices/invoices.exportBatchVoucher.spec.js [hunk 1]
```diff
@@ -8,13 +8,18 @@ import NewFund from '../../support/fragments/finance/funds/newFund';
...
+import SettingsInvoices from '../../support/fragments/invoices/settingsInvoices';
 
 describe('ui-invoices-settings: Export batch voucher', () => {
   const invoice = { ...NewInvoice.defaultUiInvoice };
   const vendorPrimaryAddress = { ...VendorAddress.vendorAddress };
   const invoiceLine = { ...NewInvoiceLine.defaultUiInvoiceLine };
   const fund = { ...NewFund.defaultFund };
   const subtotalValue = 100;
+  const batchGroupConfiguration = {
+    batchGroupId: 'cd592659-77aa-4eb3-ac34-c9a4657bb20f',
```
Comment (MaksimTerekhov) [inline cypress/integration/invoices/invoices.exportBatchVoucher.spec.js:20]: Seems that is hardcode. It is possible to genereate it with uuid()?

File: cypress/support/fragments/inventory/inventorySearch.js [hunk 2]
```diff
@@ -175,6 +175,12 @@ export default {
     cy.do(Button('Search').click());
     // cy.wait(['@getHoldings', '@getProfiles']);
   },
+  simpleSearchByParameter: (parameter, value) => {
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/inventory/inventorySearch.js:178]: May be just searchByParameter without simple?

**Review by kimkate303 — APPROVED** [review]

File: cypress/integration/invoices/invoices.exportBatchVoucher.spec.js (hunk 1 above)
Comment (IhorBohdan) [reply cypress/integration/invoices/invoices.exportBatchVoucher.spec.js]: Done

File: cypress/support/fragments/inventory/inventorySearch.js (hunk 2 above)
Comment (IhorBohdan) [reply cypress/support/fragments/inventory/inventorySearch.js:178]: searchByParametr with some parametrs in that method explore by Tanya and Dima.

File: cypress/integration/invoices/invoices.exportBatchVoucher.spec.js [hunk 3]
```diff
@@ -25,7 +30,12 @@ describe('ui-invoices-settings: Export batch voucher', () => {
           organization.addresses.find(address => address.isPrimary === true));
       });
     cy.getBatchGroups()
-      .then(batchGroup => { invoice.batchGroup = batchGroup.name; });
+      .then(batchGroup => {
+        invoice.batchGroup = batchGroup.name;
+        batchGroupConfiguration.batchGroupId = batchGroup.id;
+      });
+    console.log(batchGroupConfiguration);
```
Comment (dmitriyfolio) [inline cypress/integration/invoices/invoices.exportBatchVoucher.spec.js:37]: do you need this logging in final code?

File: cypress/support/fragments/inventory/inventorySearch.js (hunk 2 above)
Comment (dmitriyfolio) [reply cypress/support/fragments/inventory/inventorySearch.js:178]: does it mean that there are several methods with nearest steps? If yes, can we join them into one method?

File: cypress/support/fragments/inventory/inventorySearch.js [hunk 4]
```diff
@@ -175,6 +175,12 @@ export default {
...
+    cy.do(SearchField({ id: 'input-inventory-search' }).selectIndex(parameter));
+    cy.do(TextField('Search ').fillIn(value));
+    cy.do(Button('Search').focus());
```
Comment (dmitriyfolio) [inline cypress/support/fragments/inventory/inventorySearch.js:181]: move Search button to fragment const

File: cypress/support/fragments/invoices/settingsInvoices.js [hunk 5]
```diff
@@ -92,5 +92,13 @@ export default {
       Selection({ name: 'format' }).open(),
       SelectionList().select('JSON'),
     ]);
+  },
+
+  setConfigurationBatchGroup: (body) => {
```
Comment (dmitriyfolio) [inline cypress/support/fragments/invoices/settingsInvoices.js:97]: excess {}

Comment (IhorBohdan) [reply cypress/support/fragments/invoices/settingsInvoices.js:97]: Don't understand

File: cypress/support/fragments/inventory/inventorySearch.js (hunk 4 above)
Comment (IhorBohdan) [reply cypress/support/fragments/inventory/inventorySearch.js]: done

File: cypress/integration/invoices/invoices.exportBatchVoucher.spec.js (hunk 3 above)
Comment (IhorBohdan) [reply cypress/integration/invoices/invoices.exportBatchVoucher.spec.js]: Deleted

File: cypress/support/fragments/invoices/settingsInvoices.js (hunk 5 above)
Comment (IhorBohdan) [reply cypress/support/fragments/invoices/settingsInvoices.js]: Done

**Review by dmitriyfolio — APPROVED** [review]

**Pull request merged**